src/test/resources/Room.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Confirming the functionality of the room top column
+
+  Scenario Outline: Validating the functionality of 'Room' column
+    Given The user has logged in the website
+    When The user clicks on the "Rooms" top column
+    And The user clicks on the Click to Select Dates tab
```

Comment:
clicks on the click?

Reply:
Added asterisks to clarify as that is the tab name